llvm/include/llvm/Object/OffloadBinary.h

```diff
@@ -49,6 +51,31 @@ enum ImageKind : uint16_t {
   IMG_LAST,
 };
 
+class CompressedOffloadBundle {
```

Comment:
need to be updated to support v3. and refactor clang to use this header.

Reply:
Yes.  Though I was thinking I'd update this and refactor clang to use the LLVM Offloading API in another patch.

Reply:
I am OK with that